ui/panels-plugin/src/plugins/stat-chart/StatChartPanel.tsx

```diff
@@ -14,8 +14,8 @@
 import { StatChart, StatChartData, useChartsTheme } from '@perses-dev/components';
 import { Box, Skeleton } from '@mui/material';
 import { useMemo } from 'react';
```

Comment:
I noticed we had some duplicate imports while I was fixing some of the types moving around, so I fixed that in a few spots while I was in here. I'm going to add an eslint rule to auto-check for dupe imports and fix the rest of them in a separate PR to avoid making this PR do too many things.